ledger/simulation/simulator_test.go

```diff
@@ -14,137 +14,33 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with go-algorand.  If not, see <https://www.gnu.org/licenses/>.
 
-package simulation_test
+package simulation
```

Comment:
As part of test refactoring, most of the tests were moved from this file to `ledger/simulation/simulation_eval_test.go`